examples/reference/panes/HoloViews.ipynb

```diff
@@ -139,24 +414,91 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pn.panel(dmap, center=True, widget_location='right_bottom')"
+    "print(dmap_panel)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Lets try to change the name of the `FloatSlider`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "float_slider = hv_panel[1][0]\n",
+    "float_slider.name=\"Frequency (updated)\"\n",
+    "float_slider"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets try to reorganize the layout"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "widgets = hv_panel[1]\n",
+    "\n",
+    "pn.Row(pn.Column(*widgets), hv_panel[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### `widget_location`\n",
+    "\n",
+    "The HoloViews pane offers options to lay out the plot and widgets in a number of preconfigured arrangements using the ``center`` and ``widget_location`` parameters.\n",
+    "\n",
     "The ``widget_location`` parameter accepts all of the following options:\n",
     "    \n",
     "    ['left', 'bottom', 'right', 'top', 'top_left', 'top_right', 'bottom_left',\n",
     "     'bottom_right', 'left_top', 'left_bottom', 'right_top', 'right_bottom']"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pn.panel(dmap, center=True, widget_location='left')"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Customizing widgets\n",
+    "#### `widget_layout`\n",
+    "\n",
+    "Lets change the widget layout to a `Row`."
```

Comment:
When I change the `widget_layout` to a row, then the widgets are not nicely aligned.

![image](https://github.com/holoviz/panel/assets/42288570/0d114a45-bf8d-404c-bb58-01e107805dbe)

If you inspect the object you can see its because some custom margin that works in a Column but not Row has been added.

```bash
Column(sizing_mode='stretch_width')
    [0] Row
        [0] FloatSlider(end=10, margin=(20, 20, 5, 20), name='frequency', start=0.1, value=0.1, width=250)
        [1] IntSlider(end=10, margin=(0, 20, 5, 20), name='amplitude', start=1, value=1, width=250)
        [2] Select(margin=(5, 20, 20, 20), name='function', options=['sin', 'cos', 'tan'], value='sin', width=250)
    [1] HoloViews(DynamicMap, center=True, widget_layout=<class 'panel.layout.base...., widget_location='top_left')
```

Reply:
Fixed.